PVTPackage/source/MultiphaseSystem/NegativeTwoPhaseMultiphaseSystem.hpp

```diff
@@ -43,6 +43,16 @@ class NegativeTwoPhaseMultiphaseSystem : public CompositionalMultiphaseSystem
                double temperature,
                std::vector< double > feed ) override;
 
+  const std::vector< NegativeTwoPhaseFlashMultiphaseSystemProperties > & getBatchMultiphaseSystemProperties() const
+  {
+    return m_batchProps;
+  }
```

Comment:
Hmm, this should be surrounded by `#if defined(GEOSX_USE_OPENMP)` too...
Probably a mistake

Reply:
I removed this